packages/main/src/SelectPopover.hbs

```diff
@@ -2,37 +2,35 @@
 	<ui5-responsive-popover
 		hide-arrow
 		_disable-initial-focus
-		content-only-on-desktop
 		placement-type="Bottom"
 		horizontal-align="Left"
 		@ui5-after-open="{{_afterOpen}}"
 		@ui5-before-open="{{_beforeOpen}}"
 		@ui5-after-close="{{_afterClose}}"
 		@keydown="{{_onkeydown}}"
 	>
-		<div slot="header" class="ui5-responsive-popover-header">
-			<div class="row">
-				<span>{{_headerTitleText}}</span>
-				<ui5-button
-					class="ui5-responsive-popover-close-btn"
-					icon="decline"
-					design="Transparent"
-					@click="{{_toggleRespPopover}}"
-				>
-				</ui5-button>
-			</div>
-			{{#if _isPhone}}
+		{{#if _isPhone}}
```

Comment:
Is there a reason to use isPhone rather than ```content-only-on-desktop```. I would prefer ```content-only-on-desktop```, because it would prevent any future issues.

Reply:
Previously the value state was inside the content which leads to visual deviations.
![image](https://user-images.githubusercontent.com/31909318/129348050-460bce8e-36c6-47d1-80d6-5b0847dd1106.png)

This one is implemented in the same way as for ui5-input.